### ISSUE TITLE
CNX "Support" link directs to wrong page

### ISSUE THREAD

**Issue by brittweinstein** [Connexions/webview#1836]
#Scenario
Clicking "Support" takes you to a now deprecated support page (http://openstax.force.com/support?l=en_US&c=Products%3ACNX).  It should take you to the new support main page at https://openstax.secure.force.com/help

Also, the alt text should be changed to "Go to CNX Support".  Currently there is a typo: "Got" instead of "Go".

#Steps to Reproduce
Go in to any book's Web View, click the support link at the top right corner of the page

* **Product**: [ ] Tutor   [ ] Concept Coach [ ] OS Web [ ] Content Production Toolkit [X] CNX

* **How many times were you able to reproduce the issue?** 
Everytime

* **Images or Screenshots Included:** [X] Yes  [ ] No  (_attach below_)

#Environment:

* **Instance**: [X] production  [ ] qa  [ ] staging  [ ] dev  [ ] content
* **OS**: [ ] Mac OSX  [ ] Windows  [ ] iOS  [ ] Android [X] All
* **Browser**: [ ] Chrome  [ ] Firefox  [ ] Safari  [ ] Opera  [ ] IE  [ ] Edge [X] All

#Impact

* **Severity**: [ ] High [X] Medium  [ ] Low
* **Frequency of Occurrence**: [X] High [ ] Medium  [ ] Low

![support.jpg](https://images.zenhubusercontent.com/5a01faba8a75884b90866af7/5efee4d7-ef6a-4140-9590-c9e14f3a59f8)

## PRs
https://github.com/Connexions/webview/pull/1853

**Comment by kerwinso** [comment]
fix needs to happen in deployment script

**Comment by kerwinso** [comment]
After deploying https://github.com/Connexions/cnx-deploy/commit/efacf7dd30a23933a5463648b8f27d739a11fc5d to qa.cnx.org, the support link is correct now. 
(Note that in Rundeck, I had to specify the sha -- I couldn't just deploy `master`, even though this commit was merged to `master`).

**Comment by kerwinso** [comment]
Working on staging.
